### ISSUE TITLE
Dev: Update the the "X" icon to 14px X 14px with a 24px tap target

### ISSUE THREAD

**Issue by Rabia2219** [hackforla/tdm-calculator#3315]
### Overview
We need to update the "X" icon tap size in order to comply with WCAG guidelines.

#### Details
Update the the "X" icon to 14px X 14px with a 24px tap target. 
- The existing close button at 9.33px and 16px tap target falls short of established usability and accessibility standards, as the icon is too small to read at a glance. 
- A 9.33px X is below the threshold where users can confidently distinguish it from surrounding UI. On a desktop interface used by developers, residents, and city planners with varying levels of digital fluency, this creates unnecessary friction at a critical moment: exiting a modal or dismissing a panel. 
   - WCAG 2.5.5 (AAA) recommends 44x44px target
   - WCAG 2.5.8 (AA, 2.2) sets the minimum at 24x24px.

See specific examples of  components that do not have the 24PX "X" icon
<details>
<summary>Modals</summary>
<img width="600" alt="Image" src="https://github.com/user-attachments/assets/bfaae1c2-3176-4bec-ae0e-5015a2c9d3f0" />
<img width="600" alt="Image" src="https://github.com/user-attachments/assets/788e35b8-7299-4212-abd6-4b06e7e759ea" />
</details>

<details>
<summary>Overlays</summary>
<img width="300" alt="Image" src="https://github.com/user-attachments/assets/fbdb957c-9836-4997-9a16-e0edb810221d" />
</details>

<details>
<summary>Tooltips</summary>
<img width="300" alt="Image" src="https://github.com/user-attachments/assets/7db3369d-1e00-4f53-9f4b-060ec39995f1" />
<br>
<img width="600" alt="Image" src="https://github.com/user-attachments/assets/f7c2b9cb-c768-45f1-a3c4-d2eaffe5be89" />
<br>
<img width="400" alt="Image" src="https://github.com/user-attachments/assets/88957c95-31de-4f63-9093-e2ce74150c5d" />
<br>
<img width="800" alt="Image" src="https://github.com/user-attachments/assets/4692e7c7-7528-450d-8697-17bfbaf9397f" />
</details>

<details>
<summary>Toasts</summary>
This example shows a completely different type of close button.
<br>
<img width="300" alt="Image" src="https://github.com/user-attachments/assets/aeb1b392-22b6-4958-97dc-e85048836cde" />
</details>

None found on FAQ page
  
### Action Items
- [x] Assign yourself to the development issue.
- [x] Move the development issue from the Prioritized Backlog to In Progress.
- [x] Investigate close buttons for consistency and implement changes.
- [ ] Create a PR, if applicable.
- [ ] Create a new slide (or edit one if it already exists) in staging and add the before and after images
- [ ] Once the PR has been approved, use it to close this issue. If there is no PR is associated with this issue, close this issue after lead signoff.

### Resources/Instructions
- Design details: [link to final comment in GitHub issue]
- Design mockups: [link to Ready for Dev section in the related Figma page]

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - ~[Staging slide, 3315](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3f2afe8206d_0_143#slide=id.g3f2afe8206d_0_143)~
- Release Deck
   - [2026-08-18 Release deck, slide 3315](https://docs.google.com/presentation/d/16-zgcjRJ6zjrAeTPZatTKdOAhuyhhqiN6bPsCsp0Huc/edit?slide=id.g3f9ae231ea1_0_0#slide=id.g3f9ae231ea1_0_0)

**Comment by Rabia2219** [comment]
@kellymsimpson 
- [x] Design to find specific examples in the application where the following components have the 14px "X" icon and add screenshots where indicated in the top of this issue.
   - Modals
   - overlays
   - tooltips
   - toasts
   - FAQ editing - possibly

**Comment by kellymsimpson** [comment]
@ExperimentsInHonesty
Screenshots have been added. None found on the FAQ page.
I also want to point out that there are different placements of these close buttons in these instances. The bounding box should be 16 pts from the opt edge and 16pts from the right edge for all instances:
<details open>
<summary>Modals</summary>
<img width="2540" height="1660" alt="Image" src="https://github.com/user-attachments/assets/bfaae1c2-3176-4bec-ae0e-5015a2c9d3f0" />
<img width="2544" height="1676" alt="Image" src="https://github.com/user-attachments/assets/788e35b8-7299-4212-abd6-4b06e7e759ea" />
</details open>

<details open>
<summary>Tooltips</summary>
<img width="300" alt="Image" src="https://github.com/user-attachments/assets/7db3369d-1e00-4f53-9f4b-060ec39995f1" />
<br>
<img width="1670" height="552" alt="Image" src="https://github.com/user-attachments/assets/f7c2b9cb-c768-45f1-a3c4-d2eaffe5be89" />
</details open>

<details open>
<summary>Toasts</summary>

This example shows a completely different type of close button.

<img width="300" alt="Image" src="https://github.com/user-attachments/assets/aeb1b392-22b6-4958-97dc-e85048836cde" />
</details open>

**Comment by Rabia2219** [comment]
@entrotech The following items still need to be fixed

#### X on tooltips on Page 3 have not been updated

<details><summary>X icon pixel 20 </summary>
<img width="350" alt="Image" src="https://github.com/user-attachments/assets/76387281-3c2f-4c23-8682-48133ed3d8f8" />
</details> 

<details><summary>Tap target 32x26</summary>
<img width="350" alt="Image" src="https://github.com/user-attachments/assets/2a477e18-4d9d-4692-bd33-a6ad5d487616" />
</details> 


#### Tap area too large

<details><summary>Tap area extends across the entire modal at the top</summary>
<img width="533" height="557" alt="Image" src="https://github.com/user-attachments/assets/df2ede98-9b44-46e0-8ef6-13cabdd35a4e" />
</details>
